assignment02/js/script.js

```diff
@@ -0,0 +1,54 @@
+$(document).ready(function(){
+var femalename1;
+femalename1= "Haley"
+var femalename1 = prompt("Please enter a female name (1 of 5)");
+//(femalename1);
+$('#female-name1').text(femalename1);
+state1 = "New York"
+var state1 = prompt("Please enter a state name");
+state1 = "New York"
+var geographical;
+geographical = "mountain"
+var geographical = prompt("Please enter a geographical loction (1 of 2)");
```

Comment:
Again  - you only need 1 geo feature. You just assign it twice. The count should be for the number of unique values you need of that type.